helm/openehr_service/lib/commands/getPatientTop3ThingsHscnDetail.js

```diff
@@ -73,8 +75,17 @@ class GetPatientTop3ThingsHscnDetailCommand {
       throw new BadRequestError(valid.error);
     }
 
-    const { top3ThingsService } = this.ctx.services;
-    const responseObj = top3ThingsService.getLatestDetailByPatientId(patientId);
+    // const { top3ThingsService } = this.ctx.services;
+    // const responseObj = top3ThingsService.getLatestDetailByPatientId(patientId);
+  
+    const { headingService } = this.ctx.services;
```

Comment:
@DmitrySolyannik is it possible to combine all this logic to a one method in top3 service?

top3ThingsService.getLatest(host, patientId) # getLatest - or something related